### ISSUE TITLE
1.9.3 dependency problem

### ISSUE THREAD

**Issue by nosnickid** [brightbox/brightbox-cli#80]
Hello,

Looks like one of the `brightbox-cli` dependencies relies on a higher version of ruby:
```
steven@steventop:~$ sudo gem install --no-ri --no-rdoc brightbox-cli
Fetching: builder-3.2.2.gem (100%)
Fetching: excon-0.45.4.gem (100%)
Fetching: formatador-0.2.5.gem (100%)
Fetching: mime-types-data-3.2015.1120.gem (100%)
ERROR:  Error installing brightbox-cli:
	mime-types-data requires Ruby version >= 2.0.
steven@steventop:~$ ruby -v
ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]
```

I'm using ubuntu 14.04.  It's got a ruby2.0 package so I've rbenv switched to 2.0 for the moment.

**Comment by tokengeek** [comment]
HI @nosnickid 

Thanks for reporting.

You are correct: `mime-types` has just reached 3.0 and dropped Ruby 1.9.3 support. https://github.com/mime-types/ruby-mime-types/commit/026ceeaab6cf6d4ca2e6a06a15affca86023ed4d

I'll update the CLI gemspec to be pessimistic with version `mime-types` 2.0 and arrange deployment of a new release.

**Comment by tokengeek** [comment]
This issue should be resolved in v2.3.4 (https://rubygems.org/gems/brightbox-cli/versions/2.3.4)

I've installed and tested some commands with 1.9.3 locally and things look okay.

Sorry for the inconvenience.

**Comment by nosnickid** [comment]
Thanks @tokengeek
